src/pages/beta/connect.tsx

```diff
@@ -0,0 +1,49 @@
+import { useRouter } from "next/router";
+import { useEffect } from "react";
+
+import { LOCAL_STORAGE } from "src/constants/skylark";
+import { useConnectedToSkylark } from "src/hooks/useConnectedToSkylark";
+
+export default function BetaConnect() {
+  const { query, push: navigateTo } = useRouter();
+  // const { connected } = useConnectedToSkylark();
+
+  useEffect(() => {
+    if (query.uri && query.apikey) {
+      localStorage.setItem(LOCAL_STORAGE.betaAuth.uri, query.uri as string);
+      localStorage.setItem(
+        LOCAL_STORAGE.betaAuth.token,
+        query.apikey as string,
+      );
+      // storage events are not picked up in the same tab, so dispatch it for the current one
+      window.dispatchEvent(new Event("storage"));
+      navigateTo("/");
+    }
+  }, [query, navigateTo]);
+
+  console.log({ query });
+
+  return (
+    <div className="flex h-screen w-screen flex-col items-center justify-center gap-4">
+      <h1 className="mb-4 font-heading text-3xl">Skylark Auto Connect</h1>
+      {!query.uri || !query.apikey ? (
+        <>
+          <p>
+            Enter your Skylark URI and API Key into the URL to auto connect.
+          </p>
+          <p>
+            {"Format: "}{" "}
```

Comment:
can be plain text at this point right? (no `{}`)

Reply:
great spot